app/views/volunteers/index.html.erb

```diff
@@ -32,10 +32,16 @@
           <% Supervisor.decorate.each do |supervisor| %>
             <li>
               <a class="small" data-value="option1" tabIndex="-1">
+              <% if !current_user.supervisor? || supervisor == current_user %>
```

Comment:
@jmkoni yes, technically the model should already confirm that the current_user is a supervisor, so I had that at first. But without that check lots of admin-related view specs were failing, so thought maybe I needed it? I don't know enough about how the different role classes relate to each other, but my thought was OK, this is a small change that shouldn't really break other tests. I can remove it and have another look at how they behave.

Reply:
Oh actually I'm misreading this. You are good here!

Reply:
@jmkoni @metamoni FYI I want to move this kind of "is user allowed to do this?" logic out of erb and into Pundit policyfiles. I'll make a github issue about it

Reply:
@compwron would this really fit in a policy file? This isn’t really an authorization issue, rather a “should this be selected”

Reply:
It would be nice to have less logic in erb files and the policy files feel like the right place for code that evaluates what type and identity a user has. An argument could be made for putting this in a decorator instead :)